routing/missioncontrol.go

```diff
@@ -299,6 +306,25 @@ func generateBandwidthHints(sourceNode *channeldb.LightningNode,
 	return bandwidthHints, nil
 }
 
+// UpdateEdgePolicy updates the edge policy of the routing hints kept in the
+// payment session.
+func (p *paymentSession) UpdateEdgePolicy(e *channeldb.ChannelEdgePolicy) {
+
+	// run over hints and update policy if edge id and direction matches
```

Comment:
Capital at beginning of sentence. Period at end.

Reply:
Fixed